.github/workflows/release.yml

```diff
@@ -96,6 +96,7 @@ jobs:
 
   SupernovaPublish:
     runs-on: ubuntu-latest
+    environment: Publishing
```

Comment:
it might be worth moving it to the release workflow otherwise we need to approve two bits manually. Also you'll have to move the environmental variables to the publishing environment as well (not sure if you did that)

Reply:
Yep, moved the secrets to the publishing env.
Not sure I understand what you mean - it is part of the release workflow, no? or did you mean part of the Deploy job?
The thing is on iOS the release workflow is structured into multiple jobs so actually the dokka docs for example is a separate job. So there I added it as a separate job and wanted to keep it consistent on all repos so added it separately here as well. The approval is just a one-off right?

Reply:
ah cool - i thought every environment workflow will need to be approved manually, but looks like it just needs to be done once for a run, so should be good!